### PR TITLE
Add diagrams for budgetcommand and storage

### LINKED ISSUE #112: Update DG with storage

### PR DESCRIPTION
**Pull request by ryan1604**
Fixes #112

### CODE REVIEW

**Review by NeoMinWei — APPROVED** [review]
LGTM
the sequence diagram could have included how the command goes gets to the execute portion e.g. from main to parser back to main

**Review by wwweert123 — APPROVED** [review]
Wow! this is a very detailed DG on the budget feature! There is even styling included! However, why is abstract class shown like this "{abstract]"

File: docs/diagrams/Storage.puml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+@startuml
+'https://plantuml.com/class-diagram
+!include style.puml
+skinparam ClassBackgroundColor STORAGE_COLOR
+
+package Storage as StoragePackage {
+Class Storage
+Class "{abstract}\nLoadData" as LoadData
+Class "{abstract}\nSaveData" as SaveData
```
Comment (wwweert123) [inline docs/diagrams/Storage.puml:9]: Thanks for changing the typo error here

**Review by wwweert123 — COMMENTED** [review]
LGTM

File: docs/images/Storage.png
Comment (NeoMinWei) [inline docs/images/Storage.png:1]: The brackets of abstract not changed in the png.
The closing bracket is ] instead of }.

Comment (NeoMinWei) [inline docs/images/Storage.png:1]: The brackets of abstract not changed in the png.
The closing bracket is ] instead of }

**Review by NeoMinWei — COMMENTED** [review]
LGTM

File: docs/diagrams/Storage.puml (hunk 1 above)
Comment (ryan1604) [reply docs/diagrams/Storage.puml:9]: welcome:D

File: docs/images/Storage.png
Comment (ryan1604) [reply docs/images/Storage.png:1]: noted will change in future prs

**Pull request merged**